### ISSUE TITLE
Tests fail (with Image::SVG::Transform < 0.04?)

### ISSUE THREAD

**Issue by eserte** [rizen/SVG-Estimate#11]
Some of my smoker machines report the following test failure for SVG-Estimate-1.0103:

```
#   Failed test 'ellipse start point, dead north'
#   at t/33_ellipse.t line 19.
#     Structures begin differing at:
#          $got->[0] = '226'
#     $expected->[0] = '126'

#   Failed test '... end point, dead north'
#   at t/33_ellipse.t line 20.
#     Structures begin differing at:
#          $got->[0] = '226'
#     $expected->[0] = '126'

#   Failed test '... min_x'
#   at t/33_ellipse.t line 23.
#          got: '186'
#     expected: '86'

#   Failed test '... max_x'
#   at t/33_ellipse.t line 24.
#          got: '226'
#     expected: '126'

#   Failed test '... min_y'
#   at t/33_ellipse.t line 25.
#          got: '196'
#     expected: '96'

#   Failed test '... max_y'
#   at t/33_ellipse.t line 26.
#          got: '216'
#     expected: '116'
# Looks like you failed 6 tests of 8.
t/33_ellipse.t ............................ 
Dubious, test returned 6 (wstat 1536, 0x600)
Failed 6/8 subtests 
```

This seems to happen only with version 0.04 of Image::SVG::Transform, according to statistical analysis:

```
****************************************************************
Regression 'mod:Image::SVG::Transform'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']          -0.0000          0.0000      -2.63
[1='eq_0.04']         1.0000          0.0000    28708144582178208.00

R^2= 1.000, N= 30, K= 2
****************************************************************
```

**Comment by eserte** [comment]
Sorry, I misread the analysis. It's the other way around: it fails if Image::SVG::Transform != 0.04 is installed. So probably you have just to adjust the PREREQ_PM version.

**Comment by perlDreamer** [comment]
Fixed in SVG::Estimate 1.0105
